tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -372,6 +372,7 @@
   "DD_CIVISIBILITY_EARLY_FLAKE_DETECTION_ENABLED": "ci_visibility_early_flake_detection_enabled",
   "DD_CIVISIBILITY_CODE_COVERAGE_COLLECTORPATH": "ci_visibility_code_coverage_collectorpath",
   "DD_CIVISIBILITY_RUM_FLUSH_WAIT_MILLIS": "ci_visibility_rum_flush_wait_millis",
+  "DD_TEST_SESSION_NAME": "test_session_name",
```

Comment:
I usually add the corresponding dd-go PR link to the PR's description so people don't complaint about it missing XD

Reply:
Good one, here it is: https://github.com/DataDog/dd-go/pull/150765